application/src/main/kotlin/com/studentcenter/weave/application/meetingTeam/util/impl/MeetingTeamInvitationServiceImpl.kt

```diff
@@ -16,23 +17,29 @@ class MeetingTeamInvitationServiceImpl(
 ) : MeetingTeamInvitationService {
 
     override fun create(teamId: UUID): MeetingTeamInvitation {
-        val invitationLink = generateInvitationLink()
+        val invitationCode = generateInvitationCode()
+        val invitationLink = generateInvitationLink(invitationCode)
 
-        val meetingTeamInvitation = MeetingTeamInvitation(
+        return MeetingTeamInvitation(
             teamId = teamId,
+            invitationCode = invitationCode,
             invitationLink = invitationLink,
-            expirationDuration = meetingTeamInvitationProperties.expireSeconds,
-        )
-
-        meetingTeamInvitationRepository.save(meetingTeamInvitation)
+            expirationDuration = meetingTeamInvitationProperties.expireSeconds.seconds,
+        ).apply {
+            meetingTeamInvitationRepository.save(this)
+        }
```

Comment:
p4) `apply`와 `also`는 모두 컨텍스트 객체를 반환해서 문제는 없지만, 보통 `apply`는 객체 자체의 변경을, `also`는 추가적인 행위를 할 때 사용해요. 여기서 `also`가 더 적절한 것 같아요
```suggestion
        ).also {
            meetingTeamInvitationRepository.save(it)
        }
```

Reply:
좋은 지적 감사합니다!!